conjur-oss/templates/NOTES.txt

```diff
@@ -37,7 +37,24 @@
         /etc/hosts file: "$SERVICE_IP  {{ $.Values.ssl.hostname }}"
 {{- end }}
 
-2. Configure Conjur
+{{ if .Values.account.create }}
+2. Retrieve Account Credentials
+
+  To retrieve the credentials for the account created in Conjur at startup, {{ .Values.account.name | quote }}
+  , use the following commands: 
+
+      export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} \
+                                         -l "app={{ template "conjur-oss.name" . }},release={{ .Release.Name }}" \
+                                         -o jsonpath="{.items[0].metadata.name}")
+      kubectl exec --namespace {{ .Release.Namespace }} \
+                   $POD_NAME \
+                   --container={{ .Chart.Name }} \
+                   -- conjurctl role retrieve-key {{ .Values.account.name }}:user:admin
+
+  Back up this key in a safe location.
+{{- else }}
+2. Retrieve account credentials
```

Comment:
The title of this section should be `Configure Conjur` or `Create Conjur Account`.